packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -175,7 +175,7 @@ const makeAuctionDriver = async (t, customTerms, params = defaultParams) => {
     return E(zoe).offer(bidInvitation, proposal, payment, harden(offerArgs));
   };
 
-  const depositCollateral = async (collateralAmount, issuerKit) => {
+  const depositCollateral = async (collateralAmount, issuerKit, limit) => {
```

Comment:
`limit` name is confusing since it contains `{goal}`. I think this is also more correct,
```suggestion
  const depositCollateral = async (collateralAmount, issuerKit, offerArgs) => {
```